custom/enikshay/tasks.py

```diff
@@ -193,9 +193,9 @@ def get_adherence_schedule_start_date(self):
         if not raw_date:
             return None
         if parse_datetime(raw_date):
-            return parse_datetime(raw_date)
+            return (parse_datetime(raw_date)).replace(tzinfo=pytz.utc)
```

Comment:
Thanks for the catch!

The `episode.adherence_schedule_start_date` comes in a string format, I thought it would be in UTC, so I added UTC tzinfo so that date comparisons between timezone-aware and timezone-naive don't break. I am changing it to IST